server/base/src/test/java/org/apache/accumulo/server/conf/SystemConfigurationTest.java

```diff
@@ -78,7 +81,7 @@ public void initMocks() {
     VersionedProperties sysProps =
         new VersionedProperties(1, Instant.now(), Map.of(GC_PORT.getKey(), "1234",
             TSERV_SCAN_MAX_OPENFILES.getKey(), "19", TABLE_BLOOM_ENABLED.getKey(), "true"));
-    expect(propStore.get(eq(sysPropKey))).andReturn(sysProps).times(2);
+    expect(propStore.get(eq(sysPropKey))).andReturn(sysProps).once();
```

Comment:
Do you know why this changed?

Reply:
It was always only ever called once. But the test didn't verify the mocks, so it didn't fail before. I fixed that by adding a method to verify the mock objects, and corrected the number of expected calls to the correct number of expected calls.